src/App.js

```diff
@@ -3,6 +3,7 @@ import "./App.css";
 import { NavBar } from "./components/NavBar/NavBar";
 import { useTheme } from "./context/theme-context";
 import { Home } from "./pages/Home/Home";
+import { TaskPage } from "./pages/TaskPage/TaskPage";
```

Comment:
In order to have clean imports: index.js can be created inside pages, all pages like taskpage,home can be exported from there. So, we dont have to explicitly import each page in different lines and instead have single line with all names imports